### DISCUSSION TITLE
nuxt3 如何分页时候如何进行数据追加

### DISCUSSION THREAD

**Discussion by wangrunze2023** [nuxt/nuxt#25376] [Questions]
### Describe the feature

I need to make a drop-down paging. How can I still render normally after overlaying data?
```
const body = computed(() => ({
    pageSize: 10,
    pageNum: pageNum.value
}))
const {refresh,data} = await useFetch('https://xxx.com/t',{
        watch: [pageNum],
        method: 'POST',
        body
})

const loadMore = () => {
    pageNum.value++;
    refresh();
};


        <article>
            <ul class="wrz_blog-article-list">
                <article-list :article-data="data.data"/>
            </ul>
            <button @click="loadMore">{{!isMore ? "加载更多" : "没有更多了～"}}</button>
        </article>

```

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by xjccc** [comment]
save your fetch data
```
const articleList = ref([])

if (pageNum.value === 1) {
   articleList.value = data.value.data
} else {
   articleList.value.push(data.value.data)
}

```
